Submitty/Submitty
# [Feature:InstructorUI] Web GUI for remove lowest

This PR adds remove lowest feature to customization page. 
Before instructors had to upload their own customization json in order to use remove lowest feature.
With the updated change Web GUI will support the remove lowest (see screenshot for more detail)

To test it below are the steps:
1. Go to grade report page -> customization page
2. drag & drop categories to bucket
3. click the pencil button
4. see remove lowest text box
5. modify them and save 
6. go to /var/local/submitty/courses/s24/sample/rainbowgrades
7. emacs customization.json
8. see that remove lowest has be correctly added to each gradeable bucket

In addition this PR changed the text box size smaller (to fix approx two digits)
Removed collapse when clicking categories (there will be following PR which would address this issue for the whole page)


Closes #9970 
![스크린샷 2024-05-25 오후 3 59 23](https://github.com/Submitty/Submitty/assets/123261952/2c9a02e7-35be-4a68-9165-4afeafb2d086)
